### DISCUSSION TITLE
Serial2 does not work

### DISCUSSION THREAD

**Discussion by azizbillah** [khoih-prog/WebServer_WT32_ETH01#13] [General]
Hello everyone

I have an issue with my wt32_eth01, the serial2 ( TXD and RXD ) does not work even with simple serial program. 
Is this common issue in wt32? because I have tried in two different board, and found some problem?

Thank you

**Comment by khoih-prog** [comment]
This library is not the correct place to ask general question about the `WT32_ETH01`. The correct place is either **Arduino or  ESP32 Forum, etc.**

Anyway, check if this Arduino Forum thread [**Serial2 not receiving bytes on wt32-eth01**](https://forum.arduino.cc/t/serial2-not-receiving-bytes-on-wt32-eth01/1038138/7) is OK for you. If not, try to Google, etc.

Sorry I have no time to test that to know if it's correct.
